src/services/terms.ts

```diff
@@ -90,16 +90,26 @@ export class TermsTransformer {
         term.altLabels,
         term.hiddenLabels,
         term.scopeNotes,
-        term.broaderTerms.map(
-          iri => this.termsMap.get(iri.value) as RelatedTerm
-        ),
-        term.narrowerTerms.map(
-          iri => this.termsMap.get(iri.value) as RelatedTerm
-        ),
-        term.relatedTerms.map(
-          iri => this.termsMap.get(iri.value) as RelatedTerm
-        )
+        term.broaderTerms.reduce(this.mapRelatedTerms, []),
+        term.narrowerTerms.reduce(this.mapRelatedTerms, []),
+        term.relatedTerms.reduce(this.mapRelatedTerms, [])
       );
     });
   }
+
+  /**
+   * Map related IRIs to their related terms, making sure to only accept complete related terms.
+   *
+   * Related terms can be incomplete because of the SPARQL query limit (see
+   * https://github.com/netwerk-digitaal-erfgoed/network-of-terms-api/issues/36).
+   */
+  private mapRelatedTerms = (acc: RelatedTerm[], iri: RDF.Term) => {
```

Comment:
What does 'acc' mean?

Reply:
Good question! I’m used to call this argument `acc`, short for `accumulator` (from doing functional programming). 

As it turns out, [even JS calls it `accumulator`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/reduce). Would renaming this to `accumulator` help? Or is `terms`/`relatedTerms` clearer?

Reply:
If you insist. 😁 Moved the reduce into the function.

Reply:
🕺